### PR TITLE
Raspberry Pi Support

### PR DESCRIPTION
**Pull request by theilmbh**
Code to interface pyoperant with the new raspberry-pi based boxes.

### CODE REVIEW

File: scripts/behave [hunk 1]
```diff
@@ -18,7 +18,7 @@ except ImportError:
 try:
     from pyoperant.local import DATAPATH
 except ImportError:
-    DATAPATH = '/home/bird/opdat'
+    DATAPATH = '/home/pi/opdat'
```
Comment (theilmbh) [inline scripts/behave]: This will break Zog

File: setup.py [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
-comedi_poll = Extension('comedi_poll',
-                    # include_dirs = ['/usr/local/include'],
-                    libraries = ['comedi'],
-                    # library_dirs = ['/usr/local/lib'],
-                    sources = ['src/comedi_poll.c'])
+#comedi_poll = Extension('comedi_poll',
+#                    # include_dirs = ['/usr/local/include'],
+#                    libraries = ['comedi'],
+#                    # library_dirs = ['/usr/local/lib'],
+#                    sources = ['src/comedi_poll.c'])
```
Comment (theilmbh) [inline setup.py]: This will break Zog

File: docs/README.rst [hunk 3]
```diff
@@ -1,5 +1,3 @@
-NOTICE!! This package has been renamed to "opyrant" and development has moved to https://github.com/opyrant/opyrant
```
Comment (neuromusic) [inline docs/README.rst]: :(

**Comment by neuromusic** [comment]
sweet!

**Pull request merged**